src/patternfly/components/Tabs/tabs.scss

```diff
@@ -0,0 +1,188 @@
+@import "../../patternfly-utilities";
+
+.pf-c-tabs-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+// Tabs
+.pf-c-tabs {
+  // Primary Tab Items
+  --pf-c-tabs__item--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-tabs__item--current--Color: var(--pf-global--active-color--100);
+  --pf-c-tabs__item--hover--BoxShadowColor: #8b8d8f;
+  --pf-c-tabs__item--hover--BoxShadow: inset 0 pf-size-prem(2) 0 0 var(--pf-c-tabs__item--hover--BoxShadowColor);
+  --pf-c-tabs__item--current--BoxShadow: inset 0 pf-size-prem(2) 0 0 var(--pf-c-tabs__item--current--Color);
+  --pf-c-tabs__item--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tabs__item--BorderColor: #ededed;
```

Comment:
should this be

```suggestion
  --pf-c-tabs__item--BorderColor: var(--pf-global--BorderColor--light-200);
```

Reply:
It should, and now I've discovered `colors.scss`. Thanks!

Would doing this for the box shadow color be legit? Right now it's set to `#8b8d8f` which isn't within `scss-variables.scss`.

    --pf-c-tabs__item--hover--BoxShadowColor: $pf-color-black-500;

Reply:
Ideally we should have a CSS var for it if we're going to use it in the component. Can you use one of `--pf-global--BorderColor--dark` or `--pf-global--BorderColor--light` instead? I wonder what similar components with this color on a hover state?

Reply:
I'm gonna go with `--pf-global--BorderColor--light` which translates to `$pf-color-black-400` if @mceledonia is okay with it - just `100` shy of the intended `#ededed` (`$pf-color-black-500`).

`--dark` would make it `$pf-color-black-600` which is probably fine too.

Reply:
Got approval from @mceledonia, good to go with `--pf-global--BorderColor--light`. 👍